pkg/agent/client/client.go

```diff
@@ -109,17 +110,26 @@ func (c *client) FetchUpdates(ctx context.Context, req *node.FetchX509SVIDReques
 			break
 		}
 		if err != nil {
-			logrus.Errorf("failed to consume entire SVID update stream: %v", err)
+			c.c.Log.Errorf("failed to consume entire SVID update stream: %v", err)
 			c.release(nodeConn)
 			return nil, err
 		}
 
 		if resp.SvidUpdate == nil {
-			logrus.Warn("empty update in SVID update stream")
+			c.c.Log.Warn("empty update in SVID update stream")
 			continue
 		}
 
 		for _, re := range resp.SvidUpdate.RegistrationEntries {
+			if err := validateRegistrationEntry(re); err != nil {
+				c.c.Log.WithFields(logrus.Fields{
+					telemetry.RegistrationID: re.EntryId,
+					telemetry.SPIFFEID:       re.SpiffeId,
+					telemetry.Selectors:      re.Selectors,
```

Comment:
Will this be rendered/formatted properly?

Reply:
Not very pretty but it gets the job done and is well-formed.